packages/unimodules-permissions-interface/ios/UMPermissionsInterface/UMPermissionsMethodsWrapper.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <UMPermissionsInterface/UMPermissionsInterface.h>
+
+@interface UMPermissionsMethodsWrapper : NSObject
```

Comment:
I feel a little bit ambivalent about this name 🤔 cc @sjchmiela 😉

Reply:
`UMPermissionsMethodsDelegate`? However, calling such static methods feels a bit like this design could be migrated to subclassing, but wasn't 😅 I would imagine `EXMediaLibrary extends UMPermissionsAwareModule`, where `UMPermissionsAwareModule` would implement `setModuleRegistry:` and export `getPermissions` and `requestPermissions` and expose `arePermissionsGranted`. Subclasses would need to implement a method returning a list of permissions required. It would also handle registering `UMPermissionsAwareModule` in `UMPermissionsInterface`

It looks a bit strange in exported modules that we on one hand use a new instance of requester, in other place pass only class, etc.

Reply:
I think, migrated to subclassing is not a good idea. In our code is several modules, in which we use more than one requester. Because of this, we export from these modules, separated methods for each permission. 
Consequently, we need more control over how we could deal with permissions (for example it will be nice if we could change the exported method name).